tensorflow/c/experimental/filesystem/modular_filesystem_test.cc

```diff
@@ -1091,7 +1110,7 @@ TEST_P(ModularFileSystemTest, TestFilesExist) {
       GTEST_SKIP() << "NewWritableFile() not supported: " << status;
   }
 
-  EXPECT_TRUE(env_->FilesExist(filenames, /*status=*/nullptr));
+  EXPECT_TRUE(env_->FilesExist(filenames, nullptr));
```

Comment:
Our coding style says to include name of argument in comments to increase readability, when the argument is a constant

Reply:
I have to comment this test when it passed because the test time is very long. So I have to remove this comment. It should not be removed when the PR is merge